File: articles/the-collective-unconscious-meaning-scope-and-controversy/go/recurrence_strength_score.go
package main

import "fmt"

func recurrenceStrength(
	clusterStability,
	weightedCooccurrence,
	crossContextRecurrence,
	alternativeExplanationStrength float64,
) float64 {
	return 0.34*clusterStability +
		0.30*weightedCooccurrence +
		0.28*crossContextRecurrence -
		0.42*alternativeExplanationStrength
}

func main() {
	score := recurrenceStrength(0.72, 0.66, 0.58, 0.44)
	fmt.Printf("Synthetic recurrence strength after alternatives: %.3f\n", score)
}

File: articles/the-collective-unconscious-meaning-scope-and-controversy/fortran/recurrence_strength_score.f90
! The Collective Unconscious: Meaning, Scope, and Controversy
! Fortran example: simple synthetic recurrence-strength score

program recurrence_strength_score_demo
  implicit none

  real :: score

  score = 0.34 * 0.72 + 0.30 * 0.66 + 0.28 * 0.58 - 0.42 * 0.44

  print *, "Synthetic recurrence strength after alternatives:", score
end program recurrence_strength_score_demo
